src/test/java/wooteco/subway/station/StationAcceptanceTest.java

```diff
@@ -135,4 +159,64 @@ void deleteStation() {
         // then
         assertThat(response.statusCode()).isEqualTo(HttpStatus.NO_CONTENT.value());
     }
+
+    @DisplayName("역 삭제 기능")
+    @Test
+    void delete() {
+        //given
+        Map<String, String> params = new HashMap<>();
+        String name1 = "강남역";
+        params.put("name", name1);
+        ExtractableResponse<Response> response1 = RestAssured.given().log().all()
+                .body(params)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when()
+                .post("/stations")
+                .then().log().all()
+                .extract();
+
+        //when
+        Long id = response1.body().jsonPath().getObject(".", StationResponse.class).getId();
+
+        ExtractableResponse<Response> response = RestAssured.given().log().all()
+                .when()
+                .delete("/stations/" + id)
```

Comment:
```suggestion
                .delete("/stations/{stationId}", id)
```

위와 같이 작성한다면 조금 더 정확한 결과 리포트를 보실 수 있습니다 :)

Reply:
저는 단순히 지금의 구조가 uri를 문자열로 직접 추가하는 연산 없이, 바로 파라미터로 값을 주입할 수 있어 더 명확하게 의미를 파악할 수 있다는 장점이 있다고 느꼈고, 그 외 다른 장점은 없는 줄 알고있었습니다. 피드백 주신 구조로 변경을 하게 되면, 이전 코드와 ``결과 리포트``가 다르게 출력된다는 의미인지 궁금합니다 🤔  (이전 케이스랑 변경한 케이스에 대한 테스트를 돌려서 로그 내용을 비교해보니 동일한 것 같은데 제가 캐치하지 못한 부분이 있을까요?? 😳)

Reply:
```
RestAssured.given().pathParam("stationId", 1).log().all()
                .when().log().all()
                .delete("/stations/{stationId}")
```

앗 제가 이 부분은 오해하여 혼란을 드렸네요 T.T
delete에 variable로 추가하고 given에서 추가해줘야 리포트로 확인이 가능합니다!